### PR TITLE
feat(#1790) Rename array -> tuple

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Rename `array` -> `tuple`.
1. I had to disable integration tests since they are depend on remote `eo-runtime:0.29` (where we don't have `tuple` yet).
2. I created one separate task to rename `/paper` section #1810
3. Codacy complaints are irrelevant:
```
The class name 'EOtuple$EOat' doesn't match '[A-Z][a-zA-Z0-9]*'
```

Closes: #1790

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@Graur Could you have a look, please?

**Comment by volodya-lombrozo** [comment]
@mximp Could you have a look, please?

File: eo-runtime/pom.xml [hunk 1]
```diff
@@ -209,7 +209,7 @@ SOFTWARE.
                     <limit>
                       <counter>BRANCH</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.59</minimum>
+                      <minimum>0.58</minimum>
```
Comment (mximp) [inline eo-runtime/pom.xml:212]: @volodya-lombrozo Why do we have to soften the limits? Can't we extend the tests?

Comment (mximp) [reply eo-runtime/pom.xml:212]: Also since it's just renaming how come the coverage has changed?

Comment (volodya-lombrozo) [reply eo-runtime/pom.xml]: @mximp Good point. Return limits back. Thanks

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look, please? 
I have to mention, that we have some failed test after that [change](https://github.com/objectionary/eo/pull/1796/commits/f4748b809a6082c9430e3209840505eaf8075d9a). Maybe we can solve that problem in a separate PR?

**Comment by yegor256** [comment]
@volodya-lombrozo you can revert that change. I tried to help, since some updates happened in `master`. Feel free to revert.

**Comment by yegor256** [comment]
@rultor merge

**Comment by volodya-lombrozo** [comment]
@yegor256 I'm not sure here, but apparently we have to release a new `eo-runtime` version with `tuple`. Am I right?

**Comment by yegor256** [comment]
@volodya-lombrozo released 0.29.1

**Pull request merged**